src/opers.c

```diff
@@ -698,7 +706,7 @@ Obj FuncAND_FLAGS (
                 CHANGED_BAG(flags2);
             }
         }
-        hash = (UInt)INT_INTOBJ(flagsX);
+        hash = (UInt)flagsX;
```

Comment:
I guess one could argue that here, we want the shift, as those zero bits contribute nothing to the hash...
Also, with this change, we are not changing the behavior of the cache ever so slightly, which can result in performance fluctuations. Hopefully irrelevant ones, but are we 100% sure about that?